indico/web/forms/fields.py

```diff
@@ -249,6 +249,21 @@ def __init__(self, *args, **kwargs):
         super(IndicoPasswordField, self).__init__(*args, **kwargs)
 
 
+class CategoryField(HiddenField):
+    """Field that lets you select a category"""
+
+    widget = JinjaWidget('forms/category_picker_widget.html')
```

Comment:
Nope. This is actually a picker, since you pick a single category. It uses the category navigator widget.